Editor/JitInspectorHelpers.cs

```diff
@@ -226,5 +227,48 @@ public static string[] GetDebugOptions()
             "alias-analysis",
             "aggressive-inlining"
         };
+
+        public static string GetTypeName(Type type)
+        {
+            if (SpecialTypeNames.ContainsKey(type))
+            {
+                return SpecialTypeNames[type];
+            }
+            else
+            {
+                return type.Name;
+            }
+        }
+
+        public static string GetHighlightColor(Type type)
+        {
+            if (SpecialTypeNames.ContainsKey(type))
+            {
+                return "#569cd6";
+            }
+            else if (type.IsValueType)
+            {
+                return ("#86c691");
+            }
+            else
+            {
+                return ("#4ec9b0");
+            }
+        }
+
+        private static readonly IReadOnlyDictionary<Type, string> SpecialTypeNames = new Dictionary<Type, string>()
```

Comment:
```suggestion
        private static readonly Dictionary<Type, string> s_SpecialTypeNames = new Dictionary<Type, string>()
```
Exposing as `IReadOnlyDictionary` carries unnecessary perf penalty

Reply:
good to know